### PR TITLE
Updated docs in relation to the new colours

### PR DESCRIPTION
**Pull request by gurcei**
...and the commands and shortcuts relating to them.

I did this late into the night, so felt like another pair of eyes ought to look over this, just in-case.

### CODE REVIEW

File: appendix-basic65.tex [hunk 1]
```diff
@@ -486,6 +490,23 @@ \subsection{BACKGROUND}
  13 &    9  &  15   &  9   & light green \\
  14 &    9  &   9   & 15   & light blue\\
  15 &   11  &  11   & 11   & light grey\\
+\hline
+ 16 &   14  &   0   &  0   & guru meditation\\
```
Comment (Impakt) [inline appendix-basic65.tex:494]: we have a colour called that? awesome. I snuck that into the error messages on the last project i worked on

File: appendix-petsciicodes.tex [hunk 2]
```diff
@@ -250,6 +250,12 @@ \section{Control codes}
 \hline
 \specialkey{CTRL} + \megakey{E} &
 Restores the colour of the cursor back to the default (white).\\
+\hline
+\specialkey{CTRL} + \megakey{D} &
+Switches the VIC-IV to colour range 0-15 (default colours). These colours can be accessed with \specialkey{CTRL} and keys \megakey{1} to \megakey{8} or \megasymbolkey and keys \megakey{1} to \megakey{8}.\\
```
Comment (Impakt) [inline appendix-petsciicodes.tex:255]: how do you feel about changing it to something like

```keys \megakey{1} to \megakey{8} or \megasymbolkey (for the first 8 colours) and keys \megakey{1} to \megakey{8} (for the remaining 8 colours)```

File: appendix-basic65.tex [hunk 3]
```diff
@@ -459,7 +459,11 @@ \subsection{BACKGROUND}
 \item [Format:] {\bf BACKGROUND colour}
 \item [Usage:] Sets the background colour
                of the screen to the argument, which must be in the
-               range of 0 to 15. (See colour table).
+               range of 0 to 255. All colours within this range are
+               customisable via the {\bf PALETTE} command, though on
+               startup, the ROM provides the first 32 preset colours
+               described in the colour table below.
+
```
Comment (Impakt) [inline appendix-basic65.tex:466]: I'd change the wording slightly. 

```... via the PALETTE command. On startup, the MEGA65 only has the first 32 colours configured, which are described in the table below.```

Do the colours go back to the default 32 on reset as well?

File: appendix-basic65.tex (hunk 1 above)
Comment (gurcei) [reply appendix-basic65.tex:494]: Hehe nice, glad it resonates with you too then ;-)

File: appendix-basic65.tex (hunk 3 above)
Comment (Impakt) [reply appendix-basic65.tex:466]: It's probably interchangeable but I prefer `only has` over `has only`

File: appendix-basic65.tex [hunk 4]
```diff
@@ -804,9 +804,9 @@ \subsection{BORDER}
 \item [Usage:] Sets the border colour
                of the screen to the argument, which must be in the
                range of 0 to 255. All colours within this range are
-               customisable via the {\bf PALETTE} command, though on
-               startup, the ROM provides the first 32 preset colours
-               described in the colour table under {\bf BACKGROUND}
+               customisable via the {\bf PALETTE} command. On
+               startup, the MEGA65 has only the first 32 colours configured,
+               which are described in the table under {\bf BACKGROUND}
```
Comment (Impakt) [inline appendix-basic65.tex:809]: under the **BACKGROUND** command on page X

File: appendix-petsciicodes.tex [hunk 5]
```diff
@@ -252,10 +252,10 @@ \section{Control codes}
 Restores the colour of the cursor back to the default (white).\\
 \hline
 \specialkey{CTRL} + \megakey{D} &
-Switches the VIC-IV to colour range 0-15 (default colours). These colours can be accessed with \specialkey{CTRL} and keys \megakey{1} to \megakey{8} or \megasymbolkey and keys \megakey{1} to \megakey{8}.\\
+Switches the VIC-IV to colour range 0-15 (default colours). These colours can be accessed with \specialkey{CTRL} and keys \megakey{1} to \megakey{8} (for the first 8 colours) or \megasymbolkey and keys \megakey{1} to \megakey{8} (for the remaining 8 colours).\\
```
Comment (Impakt) [inline appendix-petsciicodes.tex:255]: (for the first 8 colours), 

note the comma

File: appendix-basic65.tex (hunk 3 above)
Comment (gurcei) [reply appendix-basic65.tex:466]: Ok, switching over

File: appendix-basic65.tex (hunk 4 above)
Comment (gurcei) [reply appendix-basic65.tex:809]: done

**Review by Impakt — APPROVED** [review]

**Pull request merged**